### PR TITLE
Integrated with Backend, added environment variables and reorganized

### PR DESCRIPTION
**Pull request by Erik-McKelvey**

### CODE REVIEW

**Review by sispuella — DISMISSED** [review]

**Comment by sispuella** [comment]
Django Ci test failed

**Comment by jjcarrcannings** [comment]
Post requests only work if there is a trailing slash, not sure if that's intended or not.

**Comment by jjcarrcannings** [comment]
I was getting errors from the CSRF middleware even when I tested locally. Everything worked fine when I removed the CSRF middleware from the Django settings file, but that might not be something we want to do in prod. Otherwise it seems like we need to add some support for csrf tokens in the views.py response code

**Review by sispuella — DISMISSED** [review]

File: prediction_service/urls.py
Comment (samuel-barrett) [inline prediction_service/urls.py:1]: This change (among others) seems to make the "enrollment_predictions" folder redundant. That is fine, but it makes the layout less separated. We should also get rid of predict.py, maybe?

**Review by samuel-barrett — APPROVED** [review]

**Pull request merged**